src/components/Contacts/ContactDetail/ContactDetailTab/ContactDetailTab.tsx

```diff
@@ -0,0 +1,87 @@
+import {
+  Box,
+  Button,
+  makeStyles,
+  Theme,
+  ThemeProvider,
+} from '@material-ui/core';
+import { Create } from '@material-ui/icons';
+import React from 'react';
+import theme from '../../../../theme';
+import { ContactDetailQuery } from '../ContactDetail.generated';
+
+const useStyles = makeStyles((theme: Theme) => ({
```

Comment:
I would like to move toward the `styled` API as I think it's easier/cleaner to reuse on multiple components and styles are tied directly to a specific child component instead of shared across the whole component.
https://cru-main.slack.com/archives/C01P00MTREU/p1616802400006300